.github/workflows/windows-pull-request.yaml

```diff
@@ -63,12 +65,12 @@ jobs:
           cache: "pip"
           cache-dependency-path: "requirements*.txt"
 
-      - name: Install packages
-        run: |
-          python -m pip install --upgrade pip
-          pip install --upgrade --upgrade-strategy eager -e .[dev]
+            - name: Install packages
```

Comment:
Just going to flag that some of this indentation caused problems in the windows workflows. Maybe the formatting is something we can handoff to a precommit hook??

Reply:
Good catch. Just pushed a commit that hopefully resolves.